OpenSim/Actuators/TorqueActuator.h

```diff
@@ -114,18 +114,20 @@ class OSIMACTUATORS_API TorqueActuator : public ScalarActuator {
     bool getTorqueIsGlobal() const
     {   return get_torque_is_global(); }
 
-    /** Set the 'optimal_force' property. **/
+    /** %Set the 'optimal_force' property. **/
     void setOptimalForce(double optimalForce)
     {   set_optimal_force(optimalForce); }
     /** Get the current value of the 'optimal_force' property. **/
     double getOptimalForce() const override // Part of Actuator interface.
     {   return get_optimal_force(); }
 
-    /* Set the bodies to which this actuator applies torque. */
+    /** %Set the first body to which this actuator applies torque. */
```

Comment:
I guess I meant: are the bodies actually ordered in a meaningful way? "first" and "second" might imply that one body "comes first", like maybe it's closer to the root node of the tree.

I don't think it's a big deal. We use a mix of `body*.1` and `body*.A` throughout the code I think.